docs/usage/about-us.md

```diff
@@ -26,18 +26,22 @@ Next up, we have these people who help maintain parts of Renovate:
 - [@secustor](https://github.com/secustor) worked on Terraform and Helm
 - [@fgreinacher](https://github.com/fgreinacher) worked on NuGet
 - [@Turbo87](https://github.com/Turbo87) has helped in multiple areas, especially Cargo
+- [@HonkingGoose](https://github.com/HonkingGoose) works on the documentation and helps manage the community
 
 ### Valuable contributions
 
 We want to highlight these valuable contributions, even if they are one-offs.
 Some features made a lot of people happy, and efficient!
 
-- [@HonkingGoose](https://github.com/HonkingGoose) worked on the docs and was community manager
 - [@ikesyo](https://github.com/ikesyo) regularly helpful
 - [@astellingwerf](https://github.com/astellingwerf) reviews PRs
 - [@danports](https://github.com/danports) worked on the Flux manager, and other managers. Feel free to ping `@danports` for any Flux-related issue or PR
 - [@shegox](https://github.com/shegox) worked on the Go manager, and improved our docs
 - [@setchy](https://github.com/setchy) focused on Bitbucket Cloud and replacement features
+- [@cgrindel](https://github.com/cgrindel) created the `bazel-module` manager
+- [@RahulGautamSingh](https://github.com/RahulGautamSingh) refactored our code and ...
```

Comment:
@RahulGautamSingh I see you refactored a lot of our code, but don't know what to highlight extra (special feature/good bugfix).

Reply:
I think the highlight of my commits will be perf improvements after reduced cloning during update and onboarding(ongoing).

Reply:
```suggestion
- [@RahulGautamSingh](https://github.com/RahulGautamSingh) refactored a lot of code and worked on performance improvements like reduced cloning during updates and onboarding
```

How about this?